easybuild/framework/easyblock.py

```diff
@@ -2435,8 +2435,12 @@ def build_and_install_one(ecdict, init_env):
             # collect build stats
             _log.info("Collecting build stats...")
 
-            buildstats = get_build_stats(app, start_time, build_option('command_line'))
-            _log.info("Build stats: %s" % buildstats)
+            if build_option('module_only'):
```

Comment:
???

The first build generates the `build_stats` entry in the archived easyconfig.  Then you add an extension using `--skip --force`, which will extend `build_stats` with a new list item.  Then you do `--module-only` which simply retains both entries in `build_stats` without touching them.  And then you install another extension with `--skip --force`, which adds a third item to `build_stats`.  In all cases, `--job` takes the sum of the `build_time` entries of all `build_stats` list items and multiply it by two.  So where exactly do you need to know which of the builds was done using `--skip`?  Seems like I'm still missing something...